mainstyle.css

```diff
@@ -20,52 +39,89 @@ img {
 .aboutus {
     text-align: center;
     width: 100%;
-    background-color: pink;
+    background-color: #ff8cd8;
     display: block;
+    width:auto;
+    height: auto;
+    margin-top: 40px;
```

Comment:
Can you think of a way to combine all your margin declarations into a single line?